src/catalog/catalog_entry/duck_table_entry.cpp

```diff
@@ -90,7 +91,13 @@ DuckTableEntry::DuckTableEntry(Catalog &catalog, SchemaCatalogEntry &schema, Bou
 					constraint_type = IndexConstraintType::PRIMARY;
 				}
 				if (info.indexes.empty()) {
-					AddDataTableIndex(*storage, columns, unique.keys, constraint_type);
+					pair<string, BlockPointer> index_p;
+					if (unique.is_primary_key) {
+						index_p.first = "pk_" + to_string(i);
```

Comment:
What if a user creates an index named `pk_1` on this table and then drops that index (or any other conflicting name)? Will the correct index be removed?